customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/EclipseLanguageClient.java

```diff
@@ -92,7 +92,12 @@ public EclipseLanguageClient(String pathToLanguageServerPlugin, String workspace
             } else {
                 this.server = new EclipseLanguageServerFacade(pathToLanguageServerPlugin, port);
             }
-            thread.join(10000);
+            thread.join(10 * 1000);
+            if (clientSocket.get() == null) {
+                throw new IllegalStateException("EclipseLanguageServer failed to start on CLIENT_PORT " + port + ". "
+                        + "Make sure you have stopped any previous EclipseLanguageServer. "
+                        + "If not sure, you may kill all 'java' process.");
+            }
```

Comment:
@alzimmermsft 

I've added this error message. In my case, it seems to be caused by a previous instance of language server not stopped (I don't know why, I had to kill all java processes).
https://github.com/Azure/autorest.java/issues/1491#issuecomment-1175951446

Reply:
customization-base and extension-base is not packed to the tgz. I assume they are not needed at autorest.

correct me if I am wrong.